packs/data/spells.db/ashen-wind.json

```diff
@@ -26,7 +26,7 @@
             "value": {}
         },
         "description": {
-            "value": "<p>You conjure a swirling wind of ashes that blows through the area. Each creature in the area must attempt a Fortitude save or get the cloying ash in its lungs.</p>\n<hr />\n<p><strong>Success</strong> The creature is unaffected.<br /><strong>Failure</strong> The creature is @UUID[Compendium.pf2e.conditionitems.Sickened]{Sickened 1}.<br /><strong>Critical Failure</strong> The creature is @UUID[Compendium.pf2e.conditionitems.Sickened]{Sickened 2}</p>"
+            "value": "<p>You conjure a swirling wind of ashes that blows through the area. Each creature in the area must attempt a Fortitude save or get the cloying ash in its lungs.</p>\n<hr />\n<p><strong>Success</strong> The creature is unaffected.</p><p><strong>Failure</strong> The creature is @UUID[Compendium.pf2e.conditionitems.Sickened]{Sickened 1}.</p><p><strong>Critical Failure</strong> The creature is @UUID[Compendium.pf2e.conditionitems.Sickened]{Sickened 2}.</p>"
```

Comment:
Should there be newlines between the paragraph tags?

Reply:
Newlines only make a difference to the source editor in Foundry, but worth doing regardless